src/App.js

```diff
@@ -147,11 +145,11 @@ class OldApp extends Component {
     const address = await loadAddress();
 
     if (address) {
-      InteractionManager.runAfterInteractions(() => {
-        setTimeout(() => {
+      setTimeout(() => {
+        InteractionManager.runAfterInteractions(() => {
           handleReviewPromptAction(ReviewPromptAction.TimesLaunchedSinceInstall);
-        }, 10_000);
-      });
+        });
+      }, 10_000);
```

Comment:
wowow TIL that in JS `10000 === 10_000` is `true` 🤯

Reply:
Whenever I see people arguing about whether or not JavaScript is a good language on Twitter or whatever I cannot help but think about this repo: https://github.com/denysdovhan/wtfjs?tab=readme-ov-file#-examples